packages/cadl-ranch-specs/http/models/input-basic/mockapi.ts

```diff
@@ -0,0 +1,18 @@
+import { passOnSuccess, ScenarioMockApi, mockapi, json } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.Basic_Input = passOnSuccess(
+    mockapi.get("/models/basic", (req) => {
+      if (req.body && req.body["requiredString"] && req.body["requiredInt"]) {
+          return {
+            status: 200
+          };
+        } else {
+          return {
+            status: 400,
+            body: json({ code: 400, message: `Expected InputModel parameter with required fields` }),
```

Comment:
this is still not correct! read the file, it should have `error` first.

you might need open a new PR.